lib/galaxy/app.py

```diff
@@ -52,6 +52,12 @@
 from galaxy.managers.notification import NotificationManager
 from galaxy.managers.roles import RoleManager
 from galaxy.managers.session import GalaxySessionManager
+from galaxy.managers.short_term_storage import (
+    ShortTermStorageAllocator,
+    ShortTermStorageConfiguration,
+    ShortTermStorageManager,
+    ShortTermStorageMonitor,
+)
```

Comment:
I dislike this a lot - managers bring in so many dependencies. galaxy.web.short_term_storage was a nice little isolated microservice effectively and now it just mixed in with the rest of galaxy-app.

Reply:
Then I'm happy you caught this on time :)
I moved it to managers because the Manager suffix in ShortTermStorageManager suggested me to do so, but we can put it where it makes more sense. Where do you think should be the best fitting place?

Reply:
This discussion continues here https://github.com/galaxyproject/galaxy/pull/17058#discussion_r1400909897